### ISSUE TITLE
Installation instructions missing in the readme

### ISSUE THREAD

**Issue by slikts** [web-animations/web-animations-js#198]
The readme doesn't mention an npm package or give a CDN link; it just says to include `<script src="web-animations.min.js"></script>`, so I guess it means downloading the file from releases, which is a generally poor approach of managing dependencies. I had to look it up to find: https://www.npmjs.com/package/web-animations-js

**Comment by arxpoetica** [comment]
Agree. I'm scratching my head trying to figure out how to dynamically import the library, for example, using my own feature sniffing techniques.

**Comment by Martin-Pitt** [comment]
If the readme does not mention it, it might be an unofficial reupload to the npm registry. So who knows who actually maintains that and if they don't turn out to hijack your users.

**Comment by NodeGuy** [comment]
The instructions are hidden here: https://github.com/web-animations/web-animations-js/blob/dev/docs/support.md#getting-the-polyfill
